tracer/test/Datadog.Trace.Tests/SpanContextPropagatorTests.cs

```diff
@@ -3,34 +3,322 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
-using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using System.Collections.Specialized;
+using System.Globalization;
+using Datadog.Trace.Headers;
 using FluentAssertions;
+using Moq;
 using Xunit;
 
 namespace Datadog.Trace.Tests
 {
     public class SpanContextPropagatorTests
     {
+        private const ulong TraceId = 1;
+        private const ulong SpanId = 2;
+        private const SamplingPriority SamplingPriority = Trace.SamplingPriority.UserReject;
+        private const string Origin = "origin";
+
+        private delegate void Foo(string name, out string value);
```

Comment:
[nit] unused

Reply:
`Foo` 😆 
Thank you. Fixed in c9093f728360a4bbd0f89e96245f14cc70ade931.